src/handlers/testing/align/components/GradedTestCaseResultsList.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Text } from 'ink';
+import { type GradedTestCaseResult, type Grade } from '../util/models';
+
+export function GradedTestCaseResultsList(props: {
+  testCaseHashes: string[];
+  gradedTestCaseResults: GradedTestCaseResult[];
+}) {
+  return (
+    <Box flexDirection="column">
+      {props.gradedTestCaseResults.map((gradedTestCaseResult, idx) => {
+        return (
+          <GradedResult
+            key={idx}
```

Comment:
Can we use anything other than index for key here? Does it matter? Does it matter?

Reply:
no shouldn't matter since the order doesn't change